packages/grpc-js/src/resolver-dns.ts

```diff
@@ -48,7 +49,6 @@ export const DEFAULT_PORT = 443;
 const DEFAULT_MIN_TIME_BETWEEN_RESOLUTIONS_MS = 30_000;
 
 const resolveTxtPromise = util.promisify(dns.resolveTxt);
```

Comment:
This also needs to be changed, so that all resolution operations use the custom resolver.

Reply:
ah yes correct, missed that.